ui/web/tpl/DashboardView.html

```diff
@@ -15,14 +15,13 @@ <h3 class="page-header">
                         <div style="float:left; padding-left:50px;">
                             <div class="btn-group-vertical">
                                 <button type="button" class="btn btn-lg btn-danger" disabled="disabled">Error</button>
-                                <button type="button" class="btn btn-lg btn-warning" disabled="disabled">Warning</button>
-                                <button type="button" class="btn btn-lg btn-success">Normal</button>
-                                <button type="button" class="btn btn-lg btn-primary" disabled="disabled">Idle</button>
-                                <button type="button" class="btn btn-lg btn-default" disabled="disabled">Default</button>
+                                <button type="button" class="btn btn-lg btn-warning" disabled="disabled">Idle</button>
+                                <button type="button" class="btn btn-lg btn-success">Running</button>
+                                <button type="button" class="btn btn-lg btn-primary" disabled="disabled">Unused</button>
```

Comment:
bootstrap button acting strangely with a button without text. It resizes it. have to investigate further.